extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/AuthConfig.java

```diff
@@ -31,4 +31,15 @@ public interface AuthConfig {
      */
     @WithDefault("true")
     boolean proactive();
+
+    /**
+     * By default, SecurityIdentity is injectable after Quarkus has prepared CDI request context.
```

Comment:
IMHO we should just say in the first line `Propagate security identity ti support {@link SecurityIdentity} injection.` and clarify that it is required when `SecurityIdentity` must be injected in a route handler which is registered directly with the Vert.x Web router.